src/azure-cli-core/azure/cli/core/util.py

```diff
@@ -29,6 +30,12 @@
                      'Please add this certificate to the trusted CA bundle: https://github.com/Azure/azure-cli/blob/dev/doc/use_cli_effectively.md#working-behind-a-proxy. '
                      'Error detail: {}')
 
+_PROXYID_RE = re.compile(
```

Comment:
lower() before matching, and lower all strings in pattern such as resoureGroups

Reply:
I made it case insensitive by `(?i)`. https://regex101.com/r/vYpGbI/1